api/python/cellxgene_census/tests/experimental/pp/test_online.py

```diff
@@ -23,7 +23,8 @@ def matrix(m: int, n: int) -> sparse.coo_matrix:
 @pytest.mark.parametrize("stride", [101, 53])
 @pytest.mark.parametrize("n_batches", [1, 3, 11, 101])
 @pytest.mark.parametrize("m,n", [(1200, 511), (100001, 57)])
-def test_meanvar(matrix: sparse.coo_matrix, n_batches: int, stride: int) -> None:
+@pytest.mark.parametrize("ddof", [1, 100, 10000])
```

Comment:
I would add zero.  if the list gets too long, the very large ddof values are of marginal utility IMHO